SpywareFilter/sections/tracking_servers.txt

```diff
@@ -4216,6 +4216,7 @@
 ||r.popin.cc^
 ||inrecsys.popin.cc^
 ||jp-axia.m0mentum.net^
+||kr.popin.cc^
```

Comment:
it will break some sites with popin widget
https://github.com/AdguardTeam/AdguardFilters/issues/72416